Sources/Core/API/SceneEventCollection.swift

```diff
@@ -12,9 +12,17 @@ public final class SceneEventCollection: EventCollection<Scene> {
     public private(set) lazy var clicked = makeClickedEvent()
     /// Event that gets triggered when its safe area insets changed
     public private(set) lazy var safeAreaInsetsChanged = Event<Scene, Void>(object: object)
+    /// Event that gets triggered when an actor is added to the scene
+    public private(set) lazy var actorAdded = addActor()
 
     private func makeClickedEvent() -> Event<Scene, Point> {
         object?.add(ClickPlugin())
         return Event<Scene, Point>(object: object)
     }
+    
+    private func addActor() -> Event<Scene, Actor> {
```

Comment:
@z0s is this event triggered from anywhere?

I thought it might be triggered from [here](https://github.com/JohnSundell/ImagineEngine/blob/master/Sources/Core/API/Scene.swift#L111) somewhere.

```swift
scene.events.actorAdded.trigger(with: actor)
```